lib/facter/util/sssd.rb

```diff
@@ -16,7 +16,7 @@ def self.ipa_value(path)
     end
 
     def self.sssd_value(path)
-      aug_value('Sssd.lns', '/etc/sssd/sssd.conf', path)
+      aug_value('Sssd.lns', '/etc/sssd/sssd.conf', path).split(',').map(&:strip)
```

Comment:
Get an error here when there's no value in sssd.conf, e.g. I don't have `target[.="ifp"]/user_attributes`.


```
 Facter: error while resolving custom fact "sssd": undefined method `split' for nil:NilClass
backtrace:
/usr/share/foreman-installer/modules/foreman/lib/facter/util/sssd.rb:19:in `sssd_value'
```